### DISCUSSION TITLE
A2DVI v2.0 Board - HDMI to VGA adapter not working?

### DISCUSSION THREAD

**Discussion by djdoughy** [rallepalaveev/A2DVI#3] [Q&A] [answered]
I ordered a print of the 2.0 and got it assembled. HDMI works FLAWLESSLY.
But when I use an HDMI to VGA adapter to use on my CRT (I have tried many, powered and unpowered, on the CRT and a different LCD screen), I get no signal. 
Am I missing a step? Is there a jumper to be bridged? I know on the readme.md it says "The card can produce either an analog VGA or a digital HDMI signal, but not both simultaneously. To switch between modes, a firmware update is required and opening/closing connections.", but I assumed that was referring to the versions that had the VGA port on it?

**Comment by rallepalaveev** [comment]
Hi, yes, the opening/closing connections refers only to the combined VGA/HDMI boards - from v1.0 to v1.7. A2DVI.v2.0 is HDMI only, so it only accepts the HDMI firmware. In order to convert to VGA, you can use a HDMI to VGA converter. Power is supplied through the HDMI port so it can potentially power a converter.

However, using such converters is highly untested with A2DVI and potentially it is a trial and error to find out which ones would actually work. Getting the HDMI signal out of the A2DVI board has been highly customized in order for the Pico to cope with the high bit rate, so some converters may not be able to accommodate such customization.

I hope this helps.

R.P.

**Reply by djdoughy** [reply]
Thanks for the quick reply! I am finding that out that it might be certain adapters that work. For example, the SVideo adapter i have that converts to VGA for this same "purpose" of viewing my c64 on a VGA screen, the one I have was one of the only in many similar ones out there that works with the C64 video output (I went through 3 before finding the correct one with what I suppose to be the right chipset/firmware). I also went out on a limb and sacrificed a vga cable with the "pin 12 mod" people have found that fix video issues with hdmi > vga adapters. That didn't work... 
I guess the search continues. If I find the right one, I will buy another to sacrifice I suppose to get the chipset used in its creation. 
Thanks again!

**Comment by ThorstenBr** [comment]
One addition: A2DVI is actually producing a DVI signal (DVI 640x480p60), even though the PCB uses an HDMI connector for practical reasons.
HDMI basically is a protocol extension of DVI. And almost all computer displays supporting HDMI 1.0 will also accept a DVI signal. However, some modern TVs, for example, no longer support 640x480p60.

So when you select a HDMI-to-VGA converter, you may want to check if
a) if it's able to accept a plain DVI signal,
b) if it supports the low 640x480p60 resolution.

**Reply by djdoughy** [reply]
I'm going for old school. So its going on a Dell 17" Trinitron CRT. If I found the right specsheet it says it supposed to support all VGA signals (640x480 to 1920x1080, albeit squished). I ran the gambit on it with my laptop but the lowest my laptop would push was 800x600 with ranges of 59-75hz. I think I found a promising lead on an adapter, on which it lists 640x480 as an accepted input resolution. We will find out soon. Thanks for your reply and detials!

**Comment by djdoughy** [comment, marked as answer]
Searched high and low, friend found a HDMI to VGA that specifically said it can do 640x480. Ordered it, got it in last night, and IT WORKS
![image](https://github.com/user-attachments/assets/80a31461-5028-4815-b1da-378cd9303045)
Lesson learned that there is more than just a generic adapter out there, gotta look for the deets if they are provided!
Thanks yall for the help!

Oh, and here's the link to the [Amazon Listing](https://amzn.to/40egSJa)
